qemu/tests/blockdev_inc_backup_pull_mode_test.py

```diff
@@ -0,0 +1,325 @@
+import six
+import json
+import socket
+import logging
+
+from provider import backup_utils
+from provider import blockdev_base
+from provider import job_utils
+
+from provider.nbd_image_export import InternalNBDExportImage
+from provider.virt_storage.storage_admin import sp_admin
+
+from virttest import qemu_storage
+from virttest import utils_disk
+from virttest import utils_misc
+
+from avocado.utils import process
+from avocado.core import exceptions
```

Comment:
Actually, you could use` test.fail` instead of `exceptions.TestFail`

Reply:
change done